recipes/simbody/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(test_package VERSION 1.0)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
```

```suggestion
project(test_package CXX)

```

Language is a plus, version is not needed, because we will not deploy anything.
conan_basic_setup is no longer needed with those new generators